src/vs/platform/terminal/common/capabilities/capabilities.ts

```diff
@@ -158,6 +159,7 @@ export interface ICommandDetectionCapability {
 	readonly onCurrentCommandInvalidated: Event<ICommandInvalidationRequest>;
 	setCwd(value: string): void;
 	setIsWindowsPty(value: boolean): void;
+	setAliases(value: string): void;
```

Comment:
We should parse/validate the aliases inside the shell integration addon so we're cleaning up potentially invalid data asap.